rest/config.go

```diff
@@ -246,8 +246,9 @@ type DeprecatedCacheConfig struct {
 }
 
 type RevCacheConfig struct {
-	Size       *uint32 `json:"size,omitempty"`        // Maximum number of revisions to store in the revision cache
-	ShardCount *uint16 `json:"shard_count,omitempty"` // Number of shards the rev cache should be split into
+	MaxItemCount     *uint32 `json:"max_item_count,omitempty"` // Maximum number of revisions to store in the revision cache
+	MaxMemoryCountMB *uint32 `json:"max_memory_count_mb"`      // Maximum amount of memory the rev cache should consume in MB
```

Comment:
What's the behaviour when both of these are set? The lower of the two?

Reply:
Correct, both will work in tandem with one another when memory limit is set. I have added a comment to `MaxMemoryCountMB` to reflect this info, let me know if it needs work.